sqlmesh/core/engine_adapter/base.py

```diff
@@ -90,6 +90,7 @@ class EngineAdapter:
     SUPPORTS_MATERIALIZED_VIEWS = False
     SUPPORTS_MATERIALIZED_VIEW_SCHEMA = False
     SUPPORTS_CLONING = False
+    SUPPORTS_MANAGED_MODELS = False
```

Comment:
I'm wondering if this should be `SUPPORTS_MANAGED_TABLES` instead. It's mainly just used to tell pytest to skip the managed model integration tests for adapters that dont support managed models